IngestArch: Differentiate between proxies (#332)
elastic/ingest-docs: docs/en/ingest-arch/15-proxy.asciidoc

image::images/ea-fs-proxy-es.png[Image showing connections between {agent} and {es} using a proxy]

TIP: This architecture works with a variety of proxying tools to allow for more flexibility in your environment.

> Consider using <<ls-networkbridge,{ls} as proxy>> when it is feasable for your use case.

Reviewer: Small typo
```suggestion
Consider using <<ls-networkbridge,{ls} as proxy>> when it is feasible for your use case.
```
🥜 gallery comment: Can we make this tip a bit more actionable? Why might it be feasible or not? What should the user consider?

I don't think we need to make the tip lengthy. A sentence or two like you have here is enough. I'm just not sure there's enough substance in the current tip to be helpful.

I could easily be missing something. Ignore me if so.

Author: It's a bit tricky, and I'm hoping to get additional insights from @jsvd and @flexitrev.  
We like customers to use our products as much as possible, and it's easier for us to support customers when they build solutions using our products. 

This architecture allows for situations where users are heavily invested  in other vendors and products, and it's easier for them to go with what they know.

Reviewer: > We like customers to use our products as much as possible, and it's easier for us to support customers when they build solutions using our products.

Can we just recommend using Logstash as a proxy and note better support as a key benefit? Again — Feel free to ignore me if needed. Thanks!

> TIP: This architecture works with a variety of proxying tools to allow for more flexibility in your environment.
> Consider using <<ls-networkbridge,{ls} as proxy>> when it is feasable for your use case.
> {ls} and the rest of the {stack} are designed and tested to work together, and can be easier to support. 

Reviewer 2: ```suggestion
TIP: This architecture works with a variety of proxying tools to allow for more flexibility in your environment.
Consider using <<ls-networkbridge,{ls} as a proxy>> since {ls} and the rest of the {stack} are designed and tested to work together, and can be easier to support. 
```
